src/test/java/api/InstancesApiExamples.java

```diff
@@ -308,6 +308,17 @@ public void shouldReturnServerErrorIfOneOfInstancesPostedWithoutTitle() throws M
     assertEquals(postResponse.getJson().getInteger("totalRecords"), Integer.valueOf(2));
   }
 
+  @Test
+  public void shouldReturnBlockedFieldsConfig() throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
+    okapiClient.get(ApiRoot.blockedFieldsConfig(), ResponseHandler.json(getCompleted));
+    Response getResponse = getCompleted.get(5, TimeUnit.SECONDS);
+
+    JsonObject response = getResponse.getJson();
+    assertThat(response.containsKey("blockedFields"), is(true));
+    assertThat(response.getJsonArray("blockedFields"), notNullValue());
```

Comment:
Why is this only checking that the array is present, rather than it's contents?

Reply:
I added checking for contents